### ISSUE TITLE
Chart doesn't fit to container width on init, when scrollbar is visible.

### ISSUE THREAD

**Issue by Denyllon** [highcharts/highcharts#10181]
#### Expected behaviour
Initial chart width should be adjusted to container width and take into account a visibility of scrollbar.

#### Actual behaviour
Currently, a chart width fits to whole viewport value (including scrollbar)

#### Live demo with steps to reproduce
http://jsfiddle.net/BlackLabel/aru37cgx/

#### Product version
v7 and above

_Internal note:_
_Looks like regression after https://github.com/highcharts/highcharts/commit/02b559d0de3dfcc86816fc526fe9e98b9f85bb5a_

**Comment by sebastianbochan** [comment]
**Internal note**
Fixed references to correct files:
- http://jsfiddle.net/BlackLabel/xjkf1y26/

**Comment by Denyllon** [comment]
To workaround this issue, just need to call `Chart.reflow()` within `chart.events.load` function, like below: http://jsfiddle.net/BlackLabel/15h7Lpb6/

**Comment by TorsteinHonsi** [comment]
I can't see any difference between v6 and v7 in this regard - am I missing something? Tested Firefox and Chrome on the Mac.

PS: The viewport height needs to be smaller than ~800px in order to demonstrate the issue.

**Comment by Denyllon** [comment]
My mistake with marking this issue as the regression.
In both versions chart is not fit to available space (scrollbar excluded), but with one difference, namely the `container` div's width on v7 respects the visibility of scrollbar (because `overflow: hidden` is set on it, which misled me when using bisect tool).

Here is the difference with `overflow: hidden` set and :
![screenshot 2019-03-05 at 11 28 31](https://user-images.githubusercontent.com/15111955/53799040-d9b95b80-3f39-11e9-99c8-b273fb1e45db.png)
![screenshot 2019-03-05 at 11 28 06](https://user-images.githubusercontent.com/15111955/53799042-d9b95b80-3f39-11e9-9dec-2ea2298905a6.png)

and after calling reflow (v7):
![screenshot 2019-03-05 at 11 34 13](https://user-images.githubusercontent.com/15111955/53799435-a62b0100-3f3a-11e9-8b52-b4634797b615.png)
v6:
![screenshot 2019-03-05 at 11 35 21](https://user-images.githubusercontent.com/15111955/53799507-c8248380-3f3a-11e9-8b03-bc4c24ff7150.png)

In conclusion, it's not regression, but the issue exists.

**Comment by sebastianbochan** [comment]
Works properly since v8.
